modules/data/lib/models/responses/target_response.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:domain/models/target.dart';
+import 'package:domain/models/topic.dart';
+
+class TargetResponse {
+  final int id;
+  final String title;
+  final double latitude;
+  final double longitude;
+  final double radius;
+  final TopicResponse? topic;
+
+  TargetResponse(this.id, this.title, this.latitude, this.longitude,
+      this.radius, this.topic);
```

Comment:
tip: add a comma at the end of arguments and reformat code  to improve readability: 
```suggestion
  TargetResponse(
      this.id, 
      this.title, 
      this.latitude, 
      this.longitude,
      this.radius, 
      this.topic,
   );
```

Same for other constructors.

Reply:
thanks!